sops.go

```diff
@@ -464,7 +491,7 @@ type EncryptedFileEmitter interface {
 // to emit plain text files from the internal SOPS representation so that they can be
 // shown
 type PlainFileEmitter interface {
-	EmitPlainFile(TreeBranch) ([]byte, error)
+	EmitPlainFile(Tree) ([]byte, error)
```

Comment:
I think this should still just take `[]TreeBranch` (or `TreeBranches`), as it doesn't use the metadata.

Reply:
oh I see. ok